lib/administrate/search.rb

```diff
@@ -8,11 +8,20 @@ def initialize(resolver, term)
     def run
       if @term.blank?
         resource_class.all
+      elsif (@scope = search_scope)
+        resource_class.send @scope
       else
         resource_class.where(query, *search_terms)
       end
     end
 
+    def search_scope
+      if (@term[-1, 1] == ':')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.